test/integration/targets/aws_neptune_subnet_group/tasks/main.yaml

```diff
@@ -0,0 +1,284 @@
+---
+- name: set connection information for all tasks
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      #security_token: "{{ security_token }}"
```

Comment:
This should be uncommented as security token is necessary to be able to run this with STS credentials (which is how the tests in CI are run). If you aren't running with STS you can set your security_token to an empty string.